src/main/java/com/imsweb/mph/mprules/MpRuleThreeYearsApart.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2019 Information Management Services, Inc.
+ */
+package com.imsweb.mph.mprules;
+
+import com.imsweb.mph.MphComputeOptions;
+import com.imsweb.mph.MphInput;
+import com.imsweb.mph.MphRule;
+import com.imsweb.mph.MphUtils;
+import com.imsweb.mph.internal.TempRuleResult;
+import com.imsweb.mph.mpgroups.GroupUtility;
+
+public class MpRuleThreeYearsApart extends MphRule {
+
+    public MpRuleThreeYearsApart(String groupId, String step) {
+        super(groupId, step);
+        setQuestion("Are there tumors diagnosed greater than three (3) years apart?");
+        setReason("Tumors diagnosed greater than three (3) years apart are multiple primaries.");
+    }
+
+    @Override
+    public TempRuleResult apply(MphInput i1, MphInput i2, MphComputeOptions options) {
+        TempRuleResult result = new TempRuleResult();
+        int diff = GroupUtility.verifyYearsApart(i1, i2, 3);
+        if (-1 == diff) {
```

Comment:
Same comment here: "GroupUtility.verifyYearsApart" seems to return an int that has a special meaning (or maybe just -1 has a special meaning?). Clearly -1 is not the number of years apart.  If the verify method is supposed to behaves like a classic "compare" method, then this could should only check < 0, > 0 or == 0. Otherwise, please use constants to make the code more readable.